app/views/content/life-as-a-teacher/change-careers/benefits-of-changing-careers/_article.html.erb

```diff
@@ -8,6 +8,8 @@
     <p>The skills and experiences you bring from previous careers are invaluable in the classroom. From communication and public speaking, to problem solving and people management, your pupils will learn more from you than you might think.</p>
     <p>As a teacher, you can watch pupils progress as they learn, making an impact from your very first lesson.</p>
     <p><%= render 'content/shared/quotes/quote_career_change_jodie' %></p>
+    <p>You may already have valuable experience working with pupils in the classroom, such as being a teaching assistant. Your expertise working with pupils will make you a desirable candidate for teacher training, enabling you to take what you have learned so far and make a smooth transition into teaching.</p>
```

Comment:
Can we say that they would definitely be a desirable candidate? It feels a bit much to say `will` (unless this has been approved already)

Reply:
Also - should we really be guaranteeing a smooth transition or does this need softening a bit? Again, ignore if already approved